### ISSUE TITLE
Windows binaries of v0.0.104 not working

### ISSUE THREAD

**Issue by chludwig** [manusa/electronim#449]
I updated to v0.0.104 on Windows 11 using the binaries (exe and zip) and noticed freezing content in all tabs. I could click once then the current tab froze. Going to the next tab and click somewhere - same result.
I had 3 Slack and one WhatsApp tab.

I rolled back to v0.0.102 and everything's working as expected.

**Comment by manusa** [comment]
This is not happening to me.
I understand that you downloaded the binaries from the [release page](https://github.com/manusa/electronim/releases/tag/v0.0.104).
If you run the exracted `Electronim.exe` binary from the decompressed archive (.zip) in a terminal `cmd.exe`, do you get any error messages that can give us a hint of what might be going on?

Is the problem also reproducible in v0.0.103 that was also released earlier today?

I'll probably release another version later this week with the latest Electron version, maybe this will fix the issue. In the meantime, please try to provide the requested info, thanks.

**Comment by chludwig** [comment]
Yes, I downloaded from the "official" release page. I can reproduce the errors in v0.0.103 as in v0.0.104 and in both versions there are no errors shown in the Windows Terminal.
After an update to a later release, I'm still logged in to all of my Slack tabs, but when downgrading to v0.0.102, I'll have to login again. Where are things like cookies and cached files stored? Might it help to delete those or will it break other things?

Btw. is there an easy way to login to Slack? When I enter my email, a new tab opens in my default browser showing the 6 squares to enter the validation code. Then I'll have to copy that uri and add it to a new ElectronIM tab. In this tab I can enter the code and will be logged in successfully. I remember that there was an option in earlier versions to choose where to open links (in the browser or in ElectronIM).

**Comment by rzrevorg** [comment]
The version 103 and 104 do not work on Linux. There are no error notifications in the console. WhatsApp and Skype are freezing.

**Comment by manusa** [comment]
> The version 103 and 104 do not work on Linux. There are no error notifications in the console. WhatsApp and Skype are freezing.

Hi @rzrevorg 

Could you please provide more details about your system (distribution, desktop environment, how you installed ElectronIM, etc.). I use it primarily on Fedora and is working fine for me.

> Where are things like cookies and cached files stored? Might it help to delete those or will it break other things?

It depends on the platform. On Windows you can find this data in `%AppData%\electronim`.
You can delete those, but you'll loose cookies and other data that the underlying browser might have stored (the app settings are stored elsewhere).

> Btw. is there an easy way to login to Slack?

I'm working on a temp solution for this, it will be available in the next version.

**Comment by rzrevorg** [comment]
>> Could you please provide more details about your system (distribution, desktop environment, how you installed ElectronIM, etc.). I use it primarily on Fedora and is working fine for me.

Ubuntu 22.04
mate de
electronim-linux-x86_64.AppImage 
kernel 6.8.0-49-generic
session: x11

>> Where are things like cookies and cached files stored? Might it help to delete those or will it break other things?

cached files stored: /home/user/.config/electronim с
learing cache doesn't help

**Comment by jvagnet** [comment]
Same issue here:
Ubuntu 24.04, x11, electronim installed via snap.

**Comment by manusa** [comment]
I think it might have to do with this: https://github.com/electron/electron/issues/44336

Are you using the non-native spell checker?
If so, can you try to switch to the native one and see if the problem gets fixed for you?

**Comment by jvagnet** [comment]
Indeed, it looks like using the native spell checker fix the problem :+1:

**Comment by chludwig** [comment]
Yes, switching to the native spell checker avoids the 'freezing' on Windows, too.

**Comment by manusa** [comment]
Closing this issue since all of the related problems should be fixed by now.
